### PR TITLE
net/frr: add carp failover option

### PR DESCRIPTION
**Pull request by mimugmail**
As discussed with @fichtner in IRC, added carpfailover.

Franco will add the carp syshook and a config loader

### CODE REVIEW

File: net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/General.xml [hunk 1]
```diff
@@ -7,6 +7,10 @@
             <default>0</default>
             <Required>Y</Required>
         </enabled>
+        <enablecarp type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
```
Comment (fichtner) [inline net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/General.xml]: Y = model version bump, N works without. Pick one :)

**Comment by fichtner** [comment]
Merged, thanks! Script follows next week.

**Comment by mimugmail** [comment]
Yep, but @fichtner will add some stuff too ..

**Comment by fichtner** [comment]
Yep, will bump later when the functional changes land.

**Pull request merged**